tests/test-rankprofiles.C

```diff
@@ -41,6 +41,168 @@
 using namespace FFPACK;
 using namespace FFLAS;
 
+///////////////////////////////////////////////////////////////////////////////////////////
```

Comment:
Don't duplicate this code. INstead, replace each call to RankProfile by  a if (par) RankProfile (...) else pRankProfile(...)